templates/partials/nav.html

```diff
@@ -0,0 +1,19 @@
+<nav class="navigation">
+  <div>
+    <a href="/"><img class="navigation__logo" src="/dist/images/google-go.jpg"></img></a>
+  </div>
+
+  {{ if .data.authenticatedUser }}
+    <div>
+      <a class="navigation__link" href="/keyword">Keyword</a>
+      <form class="navigation__form" action="/logout" method="POST">
+        <input class="navigation__input" type="submit" value="Logout"></input>
```

Comment:
Instead of `input`, that should be a button:

```suggestion
        <button class="navigation__button" type="submit">Logout</button>
```

Reply:
Fixed in 54a708cb0c2aad215551516f779c5e54293e16f9